CONTRIBUTING.md

```diff
@@ -2,14 +2,20 @@ Contributor guide
 =================
 
 # Setup
-First. [Submodule needs to be update](https://stackoverflow.com/questions/1030169/easy-way-to-pull-latest-of-all-git-submodules)
+First. Submodule needs to be update
 ```
 git submodule update --recursive
 ```
 
 You must run a script to **select a unity versions**. Now we recommend using the `2021.3.0f1` version.  
 Please run the scripts below in the root folder.
 
+If you never set execution policy on Powershell, your execution policy would be set `Restricted`, which is default execution policy for Windows client computers. 
+You need to change policy to execute Powershell scripts.
+```
+set-executionpolicy RemoteSigned
```

Comment:
```suggestion
```powershell
set-executionpolicy RemoteSigned
```

Also, I prefer `PS > ` prefix (e.g., https://github.com/planetarium/planet-node/pull/7#discussion_r927646707), but it isn't mandatory maybe 😅

Reply:
It seemed that addressed on https://github.com/planetarium/UniLibplanet/commit/74566a5a4acafb35cc657e041ed4e14565023d59.